content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,320 @@
+   <Admonition type="info">
 
-### Database migrations and deployments
+   If this is your first time using Vercel, you will be prompted to install the Vercel app in your GitHub account.
 
-In the example you deployed, migrations are applied with the `prisma migrate deploy` command during the Vercel build (as defined in the `vercel-build` script in `package.json`).
+   </Admonition>
 
-Vercel has two environments: _preview_ and _production_, where preview deployments are for pull requests and production is for the `main` branch. It's important to keep this distinction in mind in relation to the database.
+4. Click **Create**.
 
-The `DATABASE_URL` environment variable, which you defined when you imported the project, will be available to both the preview and production environments. This can be a problem if you create a pull request containing a migration because a pull request will cause the production database schema to inadvertently change.
+5. On the **Configure Project** screen, set the `DATABASE_URL` environment variable to the connection string for your PostgreSQL database:
 
-**Therefore, we recommend having separate databases for preview and production deployments** so that pull requests containing migrations don't meddle with the production database.
+   ![Configure the project](images/300-40-deploy-to-vercel-configure-project.png)
 
-To ensure that the `DATABASE_URL` you defined is only available for production deployments, open your project settings on Vercel, and navigate to the **Environment Variables** tab:
+   The database connection string has the following format:
 
-![Vercel environment variables](https://res.cloudinary.com/prismaio/image/upload/v1628761154/docs/jIQPVvt.png)
+   ```code
+   postgresql://__USER__:__PASSWORD__@__HOST__/__DATABASE__
+   ```
 
-Click the three vertical dots on the right of the `DATABASE_URL`, and select **edit**:
+   For example, on Supabase it looks similar to:
 
-![Vercel env var options](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/XwJUdWK.png)
+   ```code
+   postgresql://janedoe:randompassword@db.projectcode.supabase.co:5432/postgres
+   ```
 
-Uncheck the **Preview** checkbox:
+6. Click **Deploy**.
 
-![Vercel env var options](https://res.cloudinary.com/prismaio/image/upload/v1628761152/docs/jqcX0tV.png)
+   Vercel will now deploy your application. The `package.json` file in the root directory of the example project uses a [`vercel-build` script](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#custom-build-step-for-node.js) to run `prisma generate && prisma migrate deploy && next build`. This generates Prisma Client, applies the migrations, and then builds the Next.js project. This ensures that the generated Prisma Client in `node_modules/@prisma/client` is available to the Next.js serverless functions.
```

Comment:
Nit: "This ensures that the generated Prisma Client in `node_modules/@prisma/client` is available to the Next.js serverless functions." is not super correct. The Client is in a different, not that important folder. It is just made available for `@prisma/client` after it has been generated.